catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,533 @@
+local id = 'USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/TRENDS/YEAR';
+local subdir = 'USGS';
+
```

Comment:
If you can, add `local version = '5';` similar to how you did on the other PR.

If not, we can always do that after the PR.

Reply:
Line 37 has the "version: '5'," (same as the non-year Trends).  I'm not seeing a "local version = '5' in the above section for either. We can look at it when you get back.